src/Vector3.h

```diff
@@ -235,6 +235,12 @@ class Vector3
 		return *this;
 	}
 
+	/** Provides a hash of a vector's contents */
+	size_t operator()(const Vector3<T> & a_Vector) const
+	{
+		return ((std::hash<T>()(a_Vector.x) ^ (std::hash<T>()(a_Vector.y) << 1)) ^ std::hash<T>()(a_Vector.z));
```

Comment:
Perhaps we should migrate it from Path.h to Vector.h so that everybody uses it. It originally belonged there. (But clang complained about something, I think?)